app/controllers/events_controller.rb

```diff
@@ -184,8 +184,8 @@ def set_event
 
     def set_associations
       @event.owner = current_user
-      @event.player_type ||= Event.player_types[:single]
-      if @event.player_type == Event.player_types[:single] || @event.type == 'Rankinglist'
+      if @event.type == 'Rankinglist'
```

Comment:
I think instead of putting this quite detailed logic into the container, you can use a before_save hook in the model, that ensure the playertype is set to single before saving a rankinglist. (Pretty sure that is what this line is for, correct?) it would also avoid the rather ugly type checking.